### PR TITLE
#1109 Adjust tooltips on radio number sets in common-properties

### PR DESCRIPTION
**Pull request by nabilIbm**
Fixes : #1109 

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss [hunk 1]
```diff
@@ -39,8 +39,9 @@
 		margin-bottom: $spacing-02;
 		margin-right: $spacing-04;
 		display: grid;
-		grid-template-columns: 1fr 8fr;
+		grid-template-columns: 1fr 11fr;
 		grid-template-rows: auto;
+		align-items: center;
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:44]: This does center it but you'll also need to add a bottom margin to the tooltip to match the bottom margin of the radio button.  I think it's $spacing-01

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss [hunk 2]
```diff
@@ -39,8 +39,9 @@
 		margin-bottom: $spacing-02;
 		margin-right: $spacing-04;
 		display: grid;
-		grid-template-columns: 1fr 8fr;
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:42]: Screen Shot 2022-07-11 at 7 36 51 AM

You might need to have another look at this.  This cause longer radio to wrap too early.  I don't think `grid-template-columns` works for this case.

**Review by matthoward366 — CHANGES_REQUESTED** [review]

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss (hunk 1 above)
Comment (nabilIbm) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:46]: Updated it.

**Review by matthoward366 — CHANGES_REQUESTED** [review]

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss [hunk 3]
```diff
@@ -69,3 +69,10 @@
 		padding-bottom: $spacing-02;
 	}
 }
+[data-id~="properties-radioInteger"] {
```
[the remark this answers is not in the document]
Comment (nabilIbm) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss]: Removed it and used flex for both radio & panel selector

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss [hunk 4]
```diff
@@ -36,28 +37,23 @@
 		}
 	}
 	.properties-radioset-panel {
-		margin-bottom: $spacing-02;
+		margin-bottom: $spacing-01;
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:40]: Why was this changed?  The comment below for $spacing-01 was only for the tooltip to align it correct.  Right now things still aren't aligned.

**Review by matthoward366 — CHANGES_REQUESTED** [review]
Longer text radio buttons don't show up correctly.
Screen Shot 2022-07-19 at 3 57 09 PM

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss (hunk 4 above)
Comment (matthoward366) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:40]: The tooltip is slight off.
Screen Shot 2022-07-19 at 3 58 32 PM

**Comment by nabilIbm** [comment]
How do we want to show them in this case? when the label's length goes offscreen, the tooltip should stick next to the label's last element, not itself in the next line, right?

**Comment by matthoward366** [comment]
Correct.  I think it would still be lined up with the first line of text.

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss (hunk 4 above)
Comment (nabilIbm) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss]: Put the margin back and fixed 
image
image
image

**Review by matthoward366 — CHANGES_REQUESTED** [review]
Screen Shot 2022-08-03 at 2 41 31 PM
Screen Shot 2022-08-03 at 2 43 04 PM

Alignment is still off and the selection is now being cutoff.

**Comment by nabilIbm** [comment]
> 

image

image

File: canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss [hunk 5]
```diff
@@ -38,28 +39,28 @@
...
+			flex-basis: 100%;
+			width: 100%;
 		}
 		&:last-of-type {
 			margin-bottom: 0; // don't want margin when last radio button in group
 		}
-
 		.bx--radio-button-wrapper > label {
 			justify-content: left; // override carbon style to center justify radio-button labels
+			margin:  2.3px 0 0 $spacing-01;
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:63]: Why 2.3px?  We shouldn't need to use specific spacing values like this.

Comment (matthoward366) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:63]: I don't think this is needed at all.  Carbon should be setting this correctly already.

Comment (matthoward366) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss:63]: You might need to add `align-items: center;` to center the label.  Then set a margin of 1.5px on `properties-radioset-tooltip`.  That fixes the selection issue.

Comment (nabilIbm) [reply canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss]: `margin: 2.5px 0 1.5px 1.5px;` on `properties-radioset-tooltip `fixed the selection issue

**Review by matthoward366 — APPROVED** [review]
looks good
